CMakeLists.txt

```diff
@@ -3,6 +3,10 @@ cmake_minimum_required(VERSION 3.18)
 set(MACOSX_BUNDLE_GUI_IDENTIFIER "org.navitproject.navit")
 set(CMAKE_EXECUTABLE_ENABLE_EXPORTS "true")
 set(MACOSX_BUNDLE_BUNDLE_NAME "Navit")
+set(CPACK_PACKAGE_CONTACT "navit@navit-project.org")
+set(CPACK_DEB_COMPONENT_INSTALL ON)
+#set(CPACK_DEBIAN_MAPTOOL_PACKAGE_DEPENDS "")
```

Comment:
cant we just omit this line, since it is anyway commented out.

Reply:
Yes, we could, and I actually wanted to, but forgot to remove it. But it's in my opinion not really important as we should work on proper cpack deb support anyway. This implements only the bare minimum required to get a usable den for maptool

Reply:
just created an issue for this.